ssc/core.h

```diff
@@ -115,70 +115,55 @@ extern const var_info var_info_invalid;
 
 class handler_interface; // forward decl
 
-class compute_module
+class check_error : public general_error
 {
 public:
-	class log_item
-	{
-	public:
-		log_item() { }
-		log_item(int t, const std::string &s, float f=-1.0) 
-			: type(t), text(s), time(f) {  }
```

Comment:
Have you thoroughly checked the core modifications when using the SDKTool and in simulation notices?

Reply:
There shouldn't be any effects on moving the cmod error classes out of the compute_module class since the cmod errors did not use any of compute_module's methods or attributes.

Here's an example of an exec_error:

![image](https://user-images.githubusercontent.com/29387736/71045670-0c13fe00-20f3-11ea-8364-fb67482ecca2.png)